webpack/assets/javascripts/react_app/components/common/Loader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from './LoaderStyles';
+
+const Loader = ({ showContent, children }) => {
+const content = showContent ? {...children } :
+    <div className="spinner spinner-lg"></div>;
```

Comment:
imho it makes sense in case we ever change spinner definitions or what to have text next to it etc.